hw2/05-form.html

```diff
@@ -1,20 +1,96 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
     <title>05 - Form</title>
     <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="05-form.js"></script>
-  </head>
-  <body>
-    <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
-      </form>
-    </main>
-  </body>
+</head>
+<body class="bg-dark p-2 pt-5">
+<main class="card mx-auto col-lg-4 col-md-6 col-sm-12 p-2">
+    <form id="inputForm">
+        <h1 class="text-center p-2">Exercise 05 - Form</h1>
+        <div class="form-group p-2">
+            <label for="fullName" class="h6 fw-bold"> Full Name: </label>
+            <input type="text" id="fullName" class="form-control" required/>
+        </div>
+        <div class="form-group p-2">
+            <label for="email" class="h6 fw-bold"> Email </label>
+            <input type="email" id="email" class="form-control" required/>
+        </div>
+        <div class="form-group p-2">
+            <label for="registrationStatus" class="h6 fw-bold">
+                Registration Status:
+            </label>
+            <select
+                    name="registrationStatus"
+                    id="registrationStatus"
+                    class="form-select"
+            >
+                <option value="no submission" selected="selected">
+                    Choose an Option
+                </option>
```

Comment:
for this, you can have it disabled, so the user cant select it. So need for the value attribute.

Reply:
Will update this!